tools/docker/Dockerfile

```diff
@@ -145,10 +146,11 @@ ENTRYPOINT ["/bin/bash", "-c"]
 FROM firecracker-containerd-test as firecracker-containerd-naive-integ-test
 ARG FIRECRACKER_TARGET=x86_64-unknown-linux-musl
 
-COPY _submodules/firecracker/target/$FIRECRACKER_TARGET/release/firecracker /usr/local/bin/
-COPY _submodules/firecracker/target/$FIRECRACKER_TARGET/release/jailer /usr/local/bin/
+COPY _submodules/firecracker/target/$FIRECRACKER_TARGET/release/firecracker /usr/local/bin/firecracker
+COPY _submodules/firecracker/target/$FIRECRACKER_TARGET/release/jailer /usr/local/bin/jailer
```

Comment:
I think it is okay to remove this jailer, if you modify the Dockerfile in this PR.

Reply:
It doesn't hurt to leave it in there, imo. We are basically copying all of the built binaries of firecracker into the docker container's workspace, which I think makes sense.